packages/fe-mockserver-core/test/unit/v4/dataAccess.test.ts

```diff
@@ -317,6 +317,106 @@ describe('Data Access', () => {
         expect(countryData.length).toEqual(1);
         expect(countryData[0].Name).toEqual('U S A');
     });
+    describe('additional $filter tests', () => {
```

Comment:
While this test is interesting to test the result of the filtering, i would also like to see a test for the `addExpandForFilters` case (maybe indirectly just through some test in the odataRequest) to validate that the filter expressions are properly taken into account, and also to check what happens when you have select + expand + filter.

Reply:
Sounds good now